### ISSUE TITLE
Timeline/calendar option

### ISSUE THREAD

**Issue by FastusBellrune** [olivierkes/manuskript#1221]
I want to suggest a new feature.
I have a need to coordinate different POVs on different chapters, and I would find it useful if I could add a date(or hour, or time of the day) to the chapters/scenes for events that occur on the same day, so that on the Cheat Sheet, if I select the date, I can see all the chapters with the events of that specific day. It could also be useful to have the date for each scene in the Story line.
For now, I can manage by using the Plots feature, but it would be wonderful to have the time option.
I also have a made-up calendar, so being able to set one would be even better.

**Comment by obw** [comment]
This is practically the same feature Request like #155!

**Comment by FastusBellrune** [comment]
Depends. Did you mean to create a fantasy calendar when you said to add time to each Scene?

**Comment by obw** [comment]
> Depends. Did you mean to create a fantasy calendar when you said to add time to each Scene?

I'm not only write Fantasy or SF, and this is a Feature, which I have miss a lot of time!

When I could set a position in time for each scene, this would be a dream! Last time I had a little Fight in one of my Stories.

Persons followed 16, Chapters where the fight was followed 25 and 20 Locations where something happens. I helped myself with a Spreadsheet.

Round about 80k Words when I was done!

Was a real pain in the ass, to fix a little Error, when I need a Character at a Time as a Position, at the right time.

Oh, the hole fight was over in seven Minutes!

Regards

P.S: I got it right, a little 'Lola rennt!' Vibe!
P.S.S.: this Story plays in our world, but has also SF and Fantasy elements in it, but no Tim-Travel(This Hell I would not dare to write).
